tsconfig.json

```diff
@@ -11,13 +11,9 @@
     "experimentalDecorators": true,
     "importHelpers": true,
     "target": "es2015",
-    "typeRoots": [
-      "node_modules/@types"
-    ],
-    "lib": [
-      "es2018",
-      "dom"
-    ]
+    "skipLibCheck": true,
```

Comment:
This is an issue between some ionic libraries and the newer angular 13 on the parsing. They recommend using this option until they have the time to update their libs.